integ-test/src/test/java/org/opensearch/sql/ppl/MathematicalFunctionIT.java

```diff
@@ -262,6 +262,19 @@ public void testSqrt() throws IOException {
         rows(5.830951894845301));
   }
 
+  @Test
+  public void testCbrt() throws IOException {
+    JSONObject result =
+            executeQuery(
+                    String.format(
+                            "source=%s | eval f = cbrt(age) | fields f", TEST_INDEX_BANK));
+    verifySchema(result, schema("f", null, "double"));
+    verifyDataRows(result,
+            rows(3.174802103936399), rows(3.3019272488946267), rows(3.0365889718756627),
+            rows(3.2075343299958265), rows(3.3019272488946267), rows(3.391211443014167),
+            rows(3.2396118012774835));
```

Comment:
I want to call out the same concern in previous PR: please double check if floating point precision problem may happen. I don't see `rows()` handle this similar as `MatcherUtils.closeTo()` matcher.

Reply:
Feedback addressed in b60195c90a24932f36c65c9c12c869b63ab8b5de

Reply:
Thanks for asserting by closeTo below. Should we remove test cases here?

Reply:
Removed in 77de5e53e04e701e14124222614826169bf57191 commit